tests/nodeLabeller_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-var _ = Describe("Node Labeller", func() {
+var _ = Describe("[rfe_id:4352]Node Labeller", func() {
```

Comment:
isn't it missing these tags? [crit:medium][vendor:cnv-qe@redhat.com][level:component]